### DISCUSSION TITLE
Try and find the issues.

### DISCUSSION THREAD

**Discussion by keystroke3** [keystroke3/linkastream#3] [General]
When you get the time, you can go through the code and see if you can identify anything that could be causing a hangup.
If you need help understanding something don't hesitate to ask. Also, if you need to look at the logs, I can send you the archives for the last month.

**Comment by BellezaEmporium** [comment]
OK. I'm looking at the code and it seems that it's OK. I can try to host it on my main server to see if it could be a server-sided issue.

**Reply by keystroke3** [reply]
Yeah, you could try that. Tell me what you find

**Reply by BellezaEmporium** [reply]
Noted, gonna create a private repository with all the files in it, and pushing it onto main server.

**Reply by keystroke3** [reply]
I use `pm2` to run in my server. So you might also want to do that.

**Reply by BellezaEmporium** [reply]
I had to tweak things a little bit to make it "answer". Headless answers, but I can't seem to make the pages pop up.

I have a redis server that's operated by Upstash, since my server doesn't allow external modification.

**Reply by keystroke3** [reply]
The pages aren't important since most of the traffic is headless. Have you experienced any issues with headless?

**Reply by BellezaEmporium** [reply]
Well since my server is a Heroku alternative, Python and Node working together isn't "fun" for him. I'll need to make Python work for now

**Reply by keystroke3** [reply]
Ok, I will put it back up but make the cpu watchdog more aggressive.
